.github/workflows/push-chart.yml

```diff
@@ -0,0 +1,23 @@
+name: Auto trigger mch-repo
+
+on:
+  push:
+    branches:
+    - release-*
+    paths:
+    - 'stable/search-prod/**'
+
+  workflow_dispatch:
+
+jobs:
+  notify-repo:
+    runs-on: ubuntu-latest
+    steps:
+      # Sends a dispatch event to multicloudhub-repo to trigger an update
+      - name: Notify Repo
+        uses: peter-evans/repository-dispatch@v1.1.3
+        with:
+          token: ${{ secrets.HUB_REPO_TOKEN }}
```

Comment:
Where are these secrets configured?

Reply:
They're defined in https://github.com/open-cluster-management/search-chart/settings/secrets/actions. You're welcome to update the FF_TOKEN with a token of your own if you'd like, as long as it has enough privilege to push to a release branch

Reply:
I created them using the acm-installer bot account so it wouldn't need to be tied to anyone's personal account

Reply:
Thanks! Merging.